src/lib/containers/access_requirement_list/AccessRequirementList.tsx

```diff
@@ -217,6 +236,24 @@ export default function AccessRequirementList({
         return undefined
     }
   }
+
+  const requestDataStepCallback = (props:requestDataStepCallbackProps) => {
+    const {managedACTAccessRequirement, step, researchProjectId, formSubmitRequestObject} = props
+    if (managedACTAccessRequirement) {
+      // to identify which managedACTAccessRequirement we are looking at in step 1 & 2 forms
+      setAccessRequirementId(String(managedACTAccessRequirement.id))
+      // required for step 2 form
+      setManagedACTAccessRequirement(managedACTAccessRequirement)
+    }
+    if (researchProjectId) {
+      setresearchProjectId(researchProjectId)
+    }
+    if (formSubmitRequestObject) {
+      setFormSubmitRequestObject(formSubmitRequestObject)
+    }
+    setRequestDataStep(step)
+  }
```

Comment:
This function is called in the child components when we need to pass data among sibling components.